static/style.css

```diff
@@ -0,0 +1,93 @@
+#gb {
+  padding-left: 60px; /* make room for mac traffic lights */
+  -webkit-app-region: drag; /* make titlebar draggable */
+}
+
+#gbwa, /* google apps */
+#gbwa + div /* google notifications */
+ {
+  display: none;
+}
+
+#gb .gb_gd {
+  /* Compact header */
+  padding: 4px;
+}
+
+/* Remove gmail logo */
+.gb_ja:not(.gb_ka) .gb_qc .gb_Rb {
+  display: none !important;
+}
+
+/* Header */
+.gb_lc {
+  height: 32px !important;
```

Comment:
I'm assuming we need all these importants to override Gmail styles?  If not, let's remove them.

Reply:
Alas, I tried without important, but they didn't act.

Reply:
It's essential to `!important` to either overwrite or protect the value from being overwritten depending on the CSS loading order, so our CSS always has the top priority.